components.json

```diff
@@ -21,5 +21,6 @@
     "src/linkify-matrix.ts": "src/linkify-matrix.ts",
     "src/components/structures/LeftPanel.tsx": "src/components/structures/LeftPanel.tsx",
     "src/components/views/rooms/RoomList.tsx": "src/components/views/rooms/RoomList.tsx",
-    "src/components/views/rooms/RoomSublist.tsx": "src/components/views/rooms/RoomSublist.tsx"
+    "src/components/views/rooms/RoomSublist.tsx": "src/components/views/rooms/RoomSublist.tsx",
```

Comment:
just a note: appending in js/jsons this way (at the end of the block) would cause unresolvable conflicts when syncing with upstrem. I found it better to prepend on top (or the line before last) as you will not need to add `,` and it will not have line modifications (-) in the changeset only additions (+)